src/components/views/messages/ReactionsRow.js

```diff
@@ -125,10 +136,23 @@ export default class ReactionsRow extends React.PureComponent {
                 reactionEvents={events}
                 myReactionEvent={myReactionEvent}
             />;
-        });
+        }).filter(item => !!item);
+
+        let showAllLink;
+        if (items.length > MAX_ITEMS_WHEN_LIMITED && !showAll) {
+            items = items.slice(0, MAX_ITEMS_WHEN_LIMITED);
+            showAllLink = <a
+                className="mx_ReactionsRow_showAll"
+                href="#"
+                onClick={this.onShowAllClick}
+            >
+                {_t("Show all")}
+            </a>;
+        }
 
         return <div className="mx_ReactionsRow">
             {items}
+            {showAllLink}
```

Comment:
It's closer to a button than a link, would suggest a rename

Reply:
Fixed in https://github.com/matrix-org/matrix-react-sdk/pull/3152/commits/f9e158ff68cfdcc7cf2c9ebf3e7501b82286154a